payload/reggae/rules/common.d

```diff
@@ -487,29 +487,38 @@ Target staticLibrary(
     );
 }
 
-Target staticLibraryTarget(in string name, Target[] objects) @safe pure {
+Target staticLibraryTarget(in string name, Target[] objects) @safe {
     import std.path: defaultExtension;
     return Target(
         [buildPath("$builddir", defaultExtension(name, libExt))],
-        staticLibraryShellCommand,
+        staticLibraryShellCommand(),
```

Comment:
?

Reply:
Heh, personal preference (remember DConf last year?) - to make clear this needs to *run*, isn't CTFE-able (on Windows).